recipes-devtools/cuda/cuda-nvvm_12.2.140-1.bb

```diff
@@ -3,7 +3,14 @@ L4T_DEB_GROUP = "cuda-nvcc"
 
 require cuda-shared-binaries.inc
 
-MAINSUM = "c6931d80a2c823e4c0a42dd1371e7bda613be2444393a67970e32f05dd0ccfd2"
+MAINSUM = "1ce8fac08a532f9202f1395a4b2c7be78cf9976663373cf563d79e7164b3b158"
 MAINSUM:x86-64 = "7cd46649dbad60b59860f0965d92eea87f783abfe2cbd2c9b49a0c8544161e3a"
 
+do_compile:prepend() {
+    mv ${B}/usr/local/cuda-${CUDA_VERSION}/nvvm/* ${B}/usr/local/cuda-${CUDA_VERSION}/
```

Comment:
It will be better to move the entire directory but only if it exists:
```
if [ -d ${B}/usr/local/cuda-${CUDA_VERSION}/nvvm ]; then
    mv ${B}/usr/local/cuda-${CUDA_VERSION}/nvvm/* ${B}/usr/local/cuda-${CUDA_VERSION}/
    rm -rf ${B}/usr/local/cuda-${CUDA_VERSION}/nvvm
fi 
```

Reply:
I don't understand the need for these moves in the first place.  Is that fixing a problem of some kind?

Reply:
Yes, the `nvvm` is not packaged same as the other cuda packages

Reply:
But it works fine as-is already (at least for x86_64).  What problem did you encounter that you're trying to fix with this change?